.github/workflows/test.yml

```
@@ -10,6 +10,27 @@ on:
     - main
 
 jobs:
+  python-linting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install Flake8
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8
+      - name: Lint with flake8
+        working-directory: ./backend
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+          flake8 . --count --max-complexity=10  --statistics
```

Combining flake8 calls and looking not selecting specific errors to search for here.
```suggestion
          flake8 . --count --max-complexity=10  --show-source --statistics
```